src/components/input/input.ts

```diff
@@ -108,6 +107,93 @@ export class TextInput extends InputBase {
   /**
    * @private
    */
+  @Input() autocomplete: string;
+
+  /**
+   * @private
+   */
+  @Input() autocorrect: string;
+
+  /**
+   * @private
+   */
+  @Input() autocapitalize: string;
+
+  /**
+   * @private
+   */
+  get autofocus(): boolean { return this.autofocus; }
+  @Input() set autofocus(value: boolean) {
+    this.autofocus = value != null && `${value}` !== 'false';
+  }
+
+  /**
+   * @private
+   */
+  @Input() list: string = null;
+
+  /**
+   * @private
+   */
+  @Input() max: string | number = null;
+
+  /**
+   * @private
+   */
+  @Input() maxlength: number = null;
+
+  /**
+   * @private
+   */
+  @Input() min: string | number = null;
+
+  /**
+   * @private
+   */
+  @Input() minlength: number = null;
+
+  /**
+   * @private
+   */
+  get readonly(): boolean { return this.readonly; }
+  @Input() set readonly(value: boolean) {
+    this.readonly = value != null && `${value}` !== 'false';
```

Comment:
we are repeating this pattern a lot:
```
value != null && `${value}` !== 'false';
```

could you move it to an auxiliar function?